backend/src/akvo/lumen/component/emailer.clj

```diff
@@ -0,0 +1,80 @@
+(ns akvo.lumen.component.emailer
+  (:require [com.stuartsierra.component :as component]
+            [cheshire.core :as json]
+            [clj-http.client :as client]
+            [clojure.pprint :refer [pprint]]
+            [postal.core :as postal]))
+
+(defn email? [email]
+  (let [ks [:body :from :subject :to]]
+    (every? #(contains? email %) ks)))
+
+(defprotocol ISendEmail
+  (send-email [this email] "Send email"))
+
+(defrecord DevEmailer []
+
+  component/Lifecycle
+  (start [this]
+    this)
+  (stop [this]
+    this)
+
+  ISendEmail
+  (send-email [this email]
+    (assert (email? email) "Not a proper email (:body :from :subject :to)")
+    (pprint email)))
+
+(defn dev-emailer
+  "Emailer will pprint email. Included link will be https."
+  [options]
+  (->DevEmailer))
+
+(defrecord SMTPEmailer [config]
+
+  component/Lifecycle
+  (start [this]
+    this)
+  (stop [this]
+    this)
+
+  ISendEmail
+  (send-email [{config :config} email]
+    (postal/send-message config email)))
+
+(defn smtp-emailer
```

Comment:
If we don't use the `smtp-emailer`, should we remove it (and the dependency on `postal`)?